go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/Masterminds/semver v1.5.0
-	github.com/gardener/etcd-backup-restore v0.14.1
+	github.com/gardener/etcd-backup-restore v0.22.0
```

Comment:
```suggestion
	github.com/gardener/etcd-backup-restore v0.23.0
```
we can now vendor backup-restore `v0.23.0`

Reply:
Done